car_service/request.rb

```diff
@@ -0,0 +1,23 @@
+class Request
+  @@car_count = 0
+
+  attr_reader :arrive_time, :car_nr
+  attr_accessor :pickup_time, :wash_time
+
+  def initialize(arrive_time)
+    #time when the request was made
+    @arrive_time = arrive_time
```

Comment:
Why is this not `Time.now`?

Reply:
I made a change in the service  ` new_request = Request.new Time.now` and I forgot about the model.